deploy/eck-integrations/templates/eck-agent/cluster-role-binding.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This file and cluster-role.yml and service-account.yml should start with `elastic-agent` prefix I think

Maybe we can put all of them inside elastic-agent folder for simplicity and be more clear

Reply:
they are kinda already are inside the `eck-agent` folder 👍